src/widgets/auth-widget/auth-widget.tsx

```diff
@@ -0,0 +1,89 @@
+'use client';
+
+import { useState } from 'react';
+import { useTranslations } from 'next-intl';
+import { Link, usePathname, useRouter } from '@/shared/i18n/navigation';
+import { apiSignUpWithGooglePopup } from '@/shared/api/firebase/auth';
+import { mapGoogleAuthError } from '@/shared/api/firebase/map-google-error';
+
+import { SignUpForm, SignInForm } from '@/features/auth/';
+import { Button, Card, Divider, Flex, Radio, Typography } from 'antd';
+
+import { cardStyles, flexWrapperStyles } from './auth-widget.styles';
+
+const { Title, Text } = Typography;
+
+export function AuthWidget() {
+  const pathname = usePathname();
+  const router = useRouter();
+
+  const t = useTranslations('Auth');
+
+  const loginActive = pathname?.includes('/sign-in');
+  const signupActive = pathname?.includes('/sign-up');
+
+  const value = pathname.includes('/sign-in') ? 'login' : 'signup';
+
+  const [apiError, setApiError] = useState<string | null>(null);
+
+  const handleGoogleAuth = async () => {
+    try {
+      setApiError(null);
+      await apiSignUpWithGooglePopup();
+      router.push('/');
+    } catch (e) {
+      const { key } = mapGoogleAuthError(e);
+      setApiError(t(`apiErrors.${key}`));
+    }
+  };
+  return (
+    <Card
+      style={{
+        ...cardStyles,
+      }}
```

Comment:
cardStyles is object

```suggestion
      style={cardStyles}
```